.github/workflows/release-azure-pipelines.yml

```diff
@@ -335,10 +348,9 @@ extends:
           os: linux
         templateContext:
           inputs:
-          - ${{ each config in parameters.buildConfigs }}:
-            - input: pipelineArtifact
-              artifactName: azureauth-${{ parameters.version }}-${{ config.runtime }}.${{ config.archiveExt }}
-              targetPath: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-${{ config.runtime }}.${{ config.archiveExt }}
+          - input: pipelineArtifact
+            artifactName: azureauth-${{ parameters.version }}-packaged

```

Comment:
Sorry if this is a lame question. In the `package` job, we are not generating any artifacts with the name `azureauth-${{ parameters.version }}-packaged` no? We are generating `azureauth-${{ parameters.version }}-osx-arm64.tar.gz` / `azureauth-${{ parameters.version }}--win10-x64.zip`. Wouldn't this cause an error as the specified artifact name wouldn't exist?

Reply:
No that's a good question. So, in the package step, we are uploading `packaged` folder with all the packaged files (line 278). We download that entire folder here and upload individual files.

Reply:
All this because we are not allowed to directly upload compressed files

Reply:
ah okay! I thought artifact name has to be a file name. Got it!